src/settings/domain/models/ZipDownloadLimit.ts

```diff
@@ -0,0 +1,3 @@
+import { FileSize } from '../../../files/domain/models/File'
+
+export class ZipDownloadLimit extends FileSize {}
```

Comment:
Do you really need this extra `ZipDownloadLimit` class?  I assume you include it for better code readability.

Reply:
Yes, I like being able to declare my use case as `getSettingByName<ZipDownloadLimit> `instead of `getSettingByName<FileSize> `. Yep, better readability, I think it changes the useCase meaning completely